widgets/InputField/InputField.bbj

```diff
@@ -1,282 +1,149 @@
 use ::BBjWidget/BBjWidget.bbj::BBjWidget
 use ::WebKit/util/DynamicLoader.bbj::DynamicLoader
 use ::WebKit/model/Icon.bbj::Icon
-use ::WebKit/framework/LoginDialog/LoginDialogWindow.bbj::LoginDialogWindow
-use ::WebKit/widgets/InputField/IFieldReq.bbj::IFieldReq
+use ::WebKit/widgets/InputField/IValidator.bbj::IValidator
+use ::WebKit/widgets/InputField/IValidator.bbj::RepeatPasswordValidator
+use ::WebKit/util/ClientUtil.bbj::ClientUtil
 
 class public InputField extends BBjWidget implements Icon
-    field private IFieldReq fieldReq!
-    field private BBjString fieldType!
 
-    field private LoginDialogWindow dialog!
-    field private BBjChildWindow window!
-    field private BBjStaticText label!
-    field private BBjInputE inputField!
-    field private BBjEditBox passwordField!
-    field private BBjWindow subTextWnd!
-    field private BBjStaticText subText!
-
-    field private BBjImageCtrl suffixIcon!
-    field private BBjChildWindow iconWrapper!
-    field private BBjStaticText fieldIcon!
+    field protected BBjChildWindow window!
+    field protected IValidator validator!
     
-    field private Boolean isPasswordField! = 0
+    field protected BBjStaticText label!
```

Comment:
We need a field to identify the input in case dynamic form where multiple input fields will be rendered.